website/sidebars.js

```diff
@@ -687,6 +687,22 @@ const sidebarSettings = {
             "guides/best-practices/how-we-structure/5-the-rest-of-the-project",
           ],
         },
+        {
+          type: "category",
+          label: "Materializations best practices",
+          link: {
+            type: "doc",
+            id: "guides/best-practices/materializations/materializations-guide-1-guide-overview",
+          },
+          items: [
+            "guides/best-practices/materializations/materializations-guide-2-available-materializations",
```

Comment:
Compared to the "how we structure" family of URLs, these ones are pretty WET, could they just be 

`guides/best-practices/materializations/2-available-materializations`

in the same vein as `guides/best-practices/how-we-structure/2-staging`?

Reply:
ick. Can we have long keys but short slugs? Are those different docusaurus constructs? We just worked SO HARD to get all of the URLs down to nice tidy things and I am loath to let that fall apart so soon 😅

Reply:
i'll play around with it!

Reply:
also ty for moving your review! ❤️